### PR TITLE
String fix regarding the alternate beaming style of minim tremolos

### PR DESCRIPTION
**Pull request by Howard-C**
See description of this beaming style on #5612.

- "Beam style" -> "Stroke style"
- "All lines attached to stem" -> "All strokes attached to stems"

### CODE REVIEW

File: mscore/inspector/inspector_tremolo.ui [hunk 1]
```diff
@@ -150,7 +150,7 @@
         </item>
         <item>
          <property name="text">
-          <string>All lines attached to stem</string>
+          <string>All lines attached to stems</string>
```
Comment (Jojo-Schmitz) [inline mscore/inspector/inspector_tremolo.ui:153]: they are called beams elswhere

Comment (Jojo-Schmitz) [reply mscore/inspector/inspector_tremolo.ui:153]: If changing, I'd use "strokes" here and in other places of the same dialog

Comment (Jojo-Schmitz) [reply mscore/inspector/inspector_tremolo.ui:153]: Dorico calles them strokes, see https://steinberg.help/dorico_pro/v2/en/dorico/topics/notation_reference/notation_reference_tremolos_c.html

Comment (Howard-C) [reply mscore/inspector/inspector_tremolo.ui:153]: Then let's call them strokes.

**Comment by Jojo-Schmitz** [comment]
Not sufficient yet, the strings "Reset 'Beam style' value" and "Beam style" need to get changed too, to "...Stroke style..."

**Comment by Howard-C** [comment]
> Not sufficient yet, the strings "Reset 'Beam style' value" and "Beam style" need to get changed too, to "...Stroke style..."

Perhaps not, Mrs. Gould used "beam style". "Stroke" is a specific word for the "lines", it doesn't have a style. The style is beam style.

**Comment by Jojo-Schmitz** [comment]
It is either beams or lines or strokes, but not any mix.

**Comment by Howard-C** [comment]
You see, I really want to agree with you, but I cannot. Beam style and the details of beam style need to have different words.

**Comment by Jojo-Schmitz** [comment]
No. It those are called Beams all these text needs to say so.
As mentioned Dorico calls them Strokes.
You can have "Stroke style:" and settings being "Default" and "Beam-like" maybe.
But not "Beam style:" set to "... strokes...", that just doesn't make any sense

**Comment by Jojo-Schmitz** [comment]
"Beams" are what connects 8th notes and shorter.

**Comment by Howard-C** [comment]
Hmmm, I searched for them and I'm convinced ;-)

**Comment by Jojo-Schmitz** [comment]
Gould (and Dorico for that matter) need to be taken with a grain of salt though: She's British) as are the Dorico folks), so would use British English ("Behind **Bars**" as a not so subtle hint). But, citing Gould:

Diagonal lines through a stem indicate the note-Value of the repetition.
These are the tremolo **strokes**.
...
Each **stroke** is, in effect, a shorthand **beam**.
...
Design of the tremolo **stroke**
Each **stroke** is a little thinner than a **beam**

But:
The two-note tremolo is notated with **beams** of ordinary beam thickness and
centred (spelling ;-) between the stems
...
Except for minim tremolos, tremolo beams are not attached to stems. Minim
tremolos can be beamed in one of three ways:

File: mscore/inspector/inspector.cpp [hunk 2]
```diff
@@ -1052,9 +1052,9 @@ void InspectorTremolo::setElement()
             }
       // beam style setting is only appliable to minim two-note tremolo in non-TAB staves
       if (hasNonMinimTwoNoteTremolo || hasTabStaffTremolo) {
-            g.labelBeamStyle->setVisible(false);
-            g.beamStyle->setVisible(false);
-            g.resetBeamStyle->setVisible(false);
+            g.labelStrokeStyle->setVisible(false);
+            g.strokeStyle->setVisible(false);
+            g.resetStrokeStyle->setVisible(false);
```
Comment (Jojo-Schmitz) [inline mscore/inspector/inspector.cpp:1057]: Note to myself: conflict with my PR #5953

**Pull request merged**